game-core/src/test/java/games/strategy/debug/error/reporting/ErrorReportWindowAcceptanceTest.java

```diff
@@ -19,32 +17,25 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.test.common.swing.DisabledInHeadlessGraphicsEnvironment;
 import org.triplea.test.common.swing.SwingComponentWrapper;
 
 import swinglib.DialogBuilder;
 
-
 /**
  * Set up a window, enter in data for the user input text fields, click the submit button and verify
  * we have that data sent to the {@code reportHandler}.
  */
-@ExtendWith(MockitoExtension.class)
+@ExtendWith({DisabledInHeadlessGraphicsEnvironment.class, MockitoExtension.class})
```

Comment:
Not really an issue, but this annotation should be repeatable, which looks nicer IMO

Reply:
Agreed.  I saw the non-repeatable form used in `ErrorReportClientTest` and didn't even bother to check if it was repeatable.  Good call.  5c8670978